config/initializers/routes.rb

```diff
@@ -1,4 +1,12 @@
 class Exercism::Routes
```

Comment:
I like this.

Reply:
Great. I thought you would! 🙂 You're a good influence on me 😉

Reply:
It's the other way around ;)